pkgs/development/libraries/science/math/cudnn/releases.nix

```
@@ -0,0 +1,194 @@
+[
+  {
+    version = "7.4.2.24";
+    minCudaVersion = "10.0";
+    maxCudaVersion = "10.0";
+    url = "https =//developer.download.nvidia.com/compute/redist/cudnn/v7.4.2/cudnn-10.0-linux-x64-v7.4.2.24.tgz";
+    hash = "sha256-Lt/IagK1DRfojEeJVaMy5qHoF05+U6NFi06lH68C2qM="
+    ;
```

```suggestion
    hash = "sha256-Lt/IagK1DRfojEeJVaMy5qHoF05+U6NFi06lH68C2qM=";
```
applies to the other entries as well